apps/docs/src/app/core/component-docs/combobox/examples/combobox-dynamic-example.component.ts

```diff
@@ -12,9 +12,13 @@ export class ComboboxDynamicExampleComponent {
 
     dropdownValues = ['Apple', 'Banana', 'Kiwi', 'Strawberry', 'Tomato', 'Pineapple'];
 
-    selectItem(event: { item: any; index: number }) {
-        this.selected = event.item;
-        this.selectedIndex = event.index;
+    selectItem(event: string) {
+        this.dropdownValues.forEach((value, i) => {
+            if (value === event) {
+                this.selected = value;
+                this.selectedIndex = i;
```

Comment:
add `return;` to end function earlier when the match is found

Reply:
It's changed to findIndex method